src/jira.js

```diff
@@ -16,7 +16,7 @@ export default class JiraApi {
   constructor(options) {
     this.protocol = options.protocol || 'http';
     this.host = options.host;
-    this.port = options.port || 80;
+    this.port = options.port || null;
```

Comment:
Thanks for the review, will look over the stuff later.
About the default port change. As long as you use port 80 or 443 its just not needed to add this.
http://aaaa.com is the same as http://aaaa.com:80. Adding the 80 is unnecessary and the function to create the url can handle this.

Why I change this is because of oauth. Because at least jira atlassian is creating the string for comparision without the port (e.g. http://xxx.atlassian.net), while when settings this not to null, request is doing it with the port. Then the tokens doesn't match and auth fails. Toke me 2.5h to find this out and I only needed to remove the port. So that nobody falls in the same trap, the port should only be set if its a none standard port.